coral-component-table/src/scripts/Table.js

```diff
@@ -2460,6 +2460,12 @@ const Table = Decorator(class extends BaseComponent(HTMLTableElement) {
       if (this.head && this.head.sticky) {
         headerCell.setAttribute('role', 'presentation');
         headerCell._elements.content.setAttribute('role', 'columnheader');
+        if(!headerCell.children.length > 0) {
+          headerCell._elements.content.setAttribute('role', 'button');
+          if(!headerCell.firstChild) {
+            headerCell._elements.content.setAttribute('role', 'columnheader');
```

Comment:
Is this change required only when columnheader is sticky.

Reply:
The change is not required only when the columnheader is sticky, I moved the functionality so it triggers regardless of the sticky state